app/models/entry.rb

```diff
@@ -1,3 +1,4 @@
 class Entry < ApplicationRecord
   belongs_to :list
+  accepts_nested_attributes_for :list
```

Comment:
I'm not sure what you're using this for.  I think it would give you the functionality to create a list with multiple entries but that's not what we're building here?